### PR TITLE
Added some resources

### PR DESCRIPTION
**Pull request by PrashanthKumar0**
## What does this PR do?
Add resources

## For resources
* The cherno c++ series
* Tiny* series 
* Theory on CG

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- [x] Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-subjects.md [hunk 1]
```diff
@@ -301,6 +301,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [OpenGL](http://www.songho.ca/opengl/index.html) - Concepts and illustrations
 * [Ray Tracing in One Weekend](https://raytracing.github.io) - Peter Shirley (HTML)
 * [ShaderX series](http://www.realtimerendering.com/resources/shaderx/) - Wolfgang Engel
+* [Tiny Kaboom - 180 lines of C++ code](https://github.com/ssloy/tinykaboom/wiki) - Dmitry V. Sokolov (HTML)
```
[the remark this answers is not in the document]
Comment (eshellman) [reply books/free-programming-books-subjects.md:304]: More of a web page.

File: books/free-programming-books-subjects.md [hunk 2]
```diff
@@ -301,6 +301,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
...
+* [Tiny RayCaster - 486 lines of C++ code](https://github.com/ssloy/tinyraycaster/wiki) - Dmitry V. Sokolov (HTML)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:305]: almost too small to be a book, but it seems cool

File: books/free-programming-books-subjects.md [hunk 3]
```diff
@@ -301,6 +301,10 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
...
+* [Tiny RayTracer - 256 lines of C++ code](https://github.com/ssloy/tinyraytracer/wiki) - Dmitry V. Sokolov (HTML)
```
Comment (eshellman) [inline books/free-programming-books-subjects.md:306]: One thing that helps us decide if something is a book is the presence of a table of content (with multiple headings

**Review by eshellman — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md [hunk 4]
```diff
@@ -1079,6 +1080,7 @@
 ### Theory
 
 * [Automata Theory](https://online.stanford.edu/courses/soe-ycsautomata-automata-theory)
+* [Computer Graphics, Fall 2009](https://youtube.com/playlist?list=PL_w_qWAQZtAZhtzPI5pkAtcUVgmzdAP8g) (Youtube)
```
[the remark this answers is not in the document]
Comment (PrashanthKumar0) [reply courses/free-courses-en.md:1083]: Can you help me ?
I am new to contributiions .
I am unable to understand whats going on here .

File: books/free-programming-books-subjects.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (PrashanthKumar0) [reply books/free-programming-books-subjects.md:304]: I have seen many resource ( like http://infolab.stanford.edu/~ullman/focs.html
to say ) being web page which is already included so i thought its valid to add this .

File: books/free-programming-books-subjects.md (hunk 2 above)
Comment (PrashanthKumar0) [reply books/free-programming-books-subjects.md:305]: The book main aim of tiny* series is to present things in simple form .
also there are many subsections.

File: courses/free-courses-en.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (PrashanthKumar0) [reply courses/free-courses-en.md:1083]: But this thing covers only theory part (all the math) so theroy section is more relevant IMO.

File: books/free-programming-books-subjects.md (hunk 3 above)
Comment (PrashanthKumar0) [reply books/free-programming-books-subjects.md:306]: Yes it has that .

**Comment by PrashanthKumar0** [comment]
Hi @eshellman  
Sir , Please review it.

**Comment by PrashanthKumar0** [comment]
ping @eshellman !

**Pull request closed without merging**